Manual.md

```diff
@@ -725,9 +724,8 @@ Examples:
 	archs="x86_64-musl ~*-musl"
 	# Default value (all arches)
 	archs="*"
-	# Packages that do not depend on architecture-specific objects
-	archs=noarch
 	```
+Do not use noarch any longer. It is being deprecated and to be removed soon.
```

Comment:
Deprecation is an atomic action, removal takes time.   So this should read "It is deprecated and is being removed"

Reply:
Should probably nix 'any longer' as well.

Reply:
done